<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QWidget" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_photo">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="client_id">
         <property name="text">
          <string>client id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="employee_id">
         <property name="text">
          <string>employee id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="appliances">
         <property name="text">
          <string>appliances </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="description">
         <property name="text">
          <string>description </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="photo">
         <property name="text">
          <string>photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineClient_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEmployee_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineAppliances"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDescription"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePhoto"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="choose">
     <property name="text">
      <string>Choose a file...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_2">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
